defaults/main.yml

```diff
@@ -1,5 +1,36 @@
 ---
-horizon_path: /opt/stack/horizon/
+panel_src_path_os:
```

Comment:
but vars are hard to overwrite. You could do it in include role but not in host_vars and group_vars. So if we think that some variable could be overwritten we think that better place is defaults/main.